db/system_keyspace.cc

```diff
@@ -1241,6 +1259,7 @@ schema_ptr system_keyspace::role_permissions() {
         "role permissions for CassandraAuthorizer"
         );
         builder.with_version(system_keyspace::generate_schema_version(builder.uuid()));
+        builder.with_tombstone_gc_options(get_group0_tombstone_gc_options());
```

Comment:
Can all those  `with_tombstone_gc_options()` be replaced by a new static configurator (see `schema_builder::register_static_configurator`)?

Reply:
done